src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: $background-color-white;
+  border: 1px solid $border-color;
+  border-radius: 5%;
+  margin-bottom: 16px;
+
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__photo {
+    padding: 32px 19px 0;
+    margin-bottom: 40px;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: flex;
+  }
+
+  &__info {
+    @include padding-inline;
+
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    &:hover {
+      color: $hover-title-color;
+    }
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    color: $main-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__value {
+    color: $secondary-text-color;
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__sum {
+    color: $main-text-color;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__button{
+    @include padding-inline;
```

Comment:
I see why you need `&__button` wrapper
but (at least FYI) it's better to set inline padding for the entire `card` block instead (it's always the same - 16px)

Then you will be able to get rid of this wrapper

Reply:
Thanks for the advice